bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsHash.scala

```diff
@@ -59,13 +59,13 @@ object AgsHash {
     // Base Position
     Option(ctx.getTargets).foreach { t =>
       val time = Some(new java.lang.Long(when)).asGeminiOpt
-      val base = t.getBase
+      val asterism = t.getAsterism
 
       def toData(coord: GemOption[java.lang.Double]): Int =
         coord.asScalaOpt.map(_.doubleValue).##
 
-      val ra  = toData(base.getRaDegrees(time))
-      val dec = toData(base.getDecDegrees(time))
+      val ra  = toData(asterism.getRaDegrees(time))
```

Comment:
If there are just two targets, as for GHOST dual-target mode, the base position is equidistant from the two targets.  In general by default though it is the center of the smallest circle that encloses all of the targets.  That minimizes the maximum distance to any target in the asterism.